============================= test session starts =============================
platform win32 -- Python 3.8.5, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: c:\Users\Admin\Desktop\PyTest Automation\src, configfile: pytest.ini
plugins: allure-pytest-2.8.22, ast-transformer-1.0.3, forked-1.3.0, html-3.0.0, json-0.4.0, metadata-1.11.0, parallel-0.1.0, rerunfailures-9.1.1, soft-assertions-0.1.2, sugar-0.9.4, xdist-2.1.0
collected 5 items

src\test_cases\UI\open_weather_test.py xxxxx                             [100%]

============================= 5 xfailed in 39.47s =============================
